ui/src/views/Tasks/ViewTask/index.jsx

```diff
@@ -100,7 +101,7 @@ const getCachesFromTask = task =>
 @graphql(taskQuery, {
   options: props => ({
     fetchPolicy: 'network-only',
-    pollInterval: TASK_POLL_INTERVAL,
+    pollInterval: TASK_POLL_INTERVAL * 10,
```

Comment:
`TASK_POLL_INTERVAL` was updated to be 60,000ms (60s). Multiplying by 10 here brings this polling interval up to 10min. Before, this interval was 30s.

My guess is this `* 10` can be removed. And why was the `TASK_POLL_INTERVAL` increased from 30s to 60s?

Reply:
yes! Well spotted.. Oh, `10` was a debug one.. will remove.
In fact, we don't really need it much, since there is a subscribe method listening for updates. So I thought that once a minute would be more than enough, and then forgot about that multiplier. Will drop it. Thanks